lib/src/RedisChannel.cpp

```diff
@@ -12,7 +12,7 @@ using namespace sairedis;
 /**
  * @brief Get response timeout in milliseconds.
  */
-#define REDIS_ASIC_STATE_COMMAND_GETRESPONSE_TIMEOUT_MS (60*1000)
+#define REDIS_ASIC_STATE_COMMAND_GETRESPONSE_TIMEOUT_MS (5*60*1000)
```

Comment:
we would like to keep this as is, since there is scenario, where OA is requesting a lot of route updates, and then doing GET, those request are asynchronous, and syncd will be processing them much longer than 5sec, which will cause GET to timeout,
only in ASYNC mode which is default right now

Reply:
Before this change, the time out was 1 min. Processing longer than 5 sec but less than 1 min will not time out. The change is to make more waiting than 1 min before declaring "no response received" to OA. I guess increasing the time out to 5 min from 1 min actually helps the scenario mentioned.

Reply:
Ahh sorry, i reinterpreted that as 5 seconds :P